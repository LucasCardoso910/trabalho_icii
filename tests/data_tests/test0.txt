8
45
56
12
43
95
19
08
67
test0
