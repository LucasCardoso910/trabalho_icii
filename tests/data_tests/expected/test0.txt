bubble_sort:28,45
heap_sort:35,57
insert_binary_sort:16,29
insert_sort:22,36
merge_sort:15,32
quick_sort:22,29
selection_sort:28,21
shake_sort:23,45
